history/cultures/pandaren_group.txt

```diff
@@ -1,45 +1,54 @@
+# Mainland Pandaren (Sheili)
```

Comment:
These suggestions weren't implemented.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126053467-db5b01cd-df19-4b48-acd3-93c4c80466b8.png)

</details>